adafruit_hid/keycode.py

```diff
@@ -33,7 +33,7 @@ class Keycode:
     """USB HID Keycode constants.
 
     This list is modeled after the names for USB keycodes defined in
-    https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf#page=53.
```

Comment:
I pasted https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf#page=53 and it went to the document and page I expected (screenshot below). Could you check this again?

![image](https://user-images.githubusercontent.com/2847802/102721085-83600d00-42c6-11eb-87df-2b7c955b617b.png)